docs/databags.md

```diff
@@ -58,7 +62,7 @@ An Indexes Hash is part of a plaintext data bag item that defines the set of ind
 
 Roles Hash
 ----------
-A Roles Hash is a contextual (see above) Hash, part of a plaintext data bag item that defines roles for every node in a cluster, and is pointed to by the `node[:splunk][:config][:roles]` attribute (usually set in your environment).
+A Roles Hash is a contextual (see above) Hash, part of a plaintext data bag item that defines roles for every node in a cluster, and is pointed to by the `node[:splunk][:config][:roles]` attribute (usually set in your environment). A special key of 'shc_configs' is used for managing the roles on the search heads in a search head cluster.
```

Comment:
Used `shcluster` for it... as that's a common abbreviation. 879cf677a96733a571841838761c8768c778afae

Now here's a question... For the apps hash selection should we keep `deployer-apps` or make it match as well? (But also apps for the indexer cluster on the master are pulled with `master-apps`)  (It might be a 🚲🏠💥 but I figure a minor one).